### PR TITLE
Bug 1837986 - add the right color and alignment to the number of the normal tabs in compose tabs tray 

### PR DESCRIPTION
**Pull request by Alexandra-Virvara**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1837986

### CODE REVIEW

**Comment by Alexandra-Virvara** [comment]
infinite_tabs
18_not_selected
7_selected
22_selected
20_not_selected
1_selected
0_not_selected

File: fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayBanner.kt [hunk 1]
```diff
@@ -325,47 +328,100 @@ private fun generateSingleSelectBannerMenuItems(
...
+            normalTabCountTextModifier =
+                Modifier.padding(bottom = NUMBER_TABS_PADDING_BOTTOM)
+        }
+        in 10..MAX_VISIBLE_TABS -> {
+            normalTabCountText = normalTabCount.toString()
+            tabCountTextRatio = TWO_DIGITS_SIZE_RATIO
+            normalTabCountTextModifier =
+                Modifier.padding(
+                    bottom = NUMBER_TABS_PADDING_BOTTOM,
+                    end = NUMBER_TABS_PADDING_BOTTOM,
```
Comment (MozillaNoah) [inline fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayBanner.kt:359]: it looks like the original `TabCounter` is not adding end padding. is there a reason it's needed here? I'm seeing a visual bug on my device with 99
image

**Review by MozillaNoah — COMMENTED** [review]
A couple nits and questions, but this is looking good!

@Verdi can you take a look at the screenshots to confirm the coloring & alignment

**Comment by Verdi** [comment]
> A couple nits and questions, but this is looking good!
> 
> @Verdi can you take a look at the screenshots to confirm the coloring & alignment

The alignment looks right. I can't confirm the color because screenshots from my device change it but that should be in code. All the colors (light and dark) should be the same as the colors coded in the non-compose tab tray.

File: fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayBanner.kt [hunk 2]
```diff
@@ -325,43 +326,87 @@ private fun generateSingleSelectBannerMenuItems(
...
         Text(
             text = normalTabCountText,
-            modifier = normalTabCountTextModifier.align(Alignment.Center),
-            color = LocalContentColor.current,
-            fontSize = with(LocalDensity.current) { 12.dp.toSp() },
+            modifier = normalTabsTextModifier,
+            fontSize = with(LocalDensity.current) { counterTabsTextSize.toDp().toSp() },
             fontWeight = FontWeight.W700,
+            textAlign = TextAlign.Center,
+            color = normalTabsCountTextColor,
```
Comment (MozillaNoah) [inline fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayBanner.kt:409]: nit: could these parameters be reordered to match the ordering of `Text`

**Review by MozillaNoah — APPROVED** [review]
one nit but this is ready to go otherwise!!

**Pull request merged**